### PR TITLE
New Fork with Account Setting Page

### PR DESCRIPTION
**Pull request by cuchaos**
My Fork not fork the main repo so I create a new fork

### CODE REVIEW

File: Html_File/AccountSetting.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            <td>2023-01-03</td>
+            <td>Product C</td>
+            <td>$47999.00</td>
+            <td>Shipped</td>
+            <td>
+                <a href="javascript:void(0);" id="detail" type="button">Open</a>
+            </td>            
+          </tr>
+        </tbody>
+      </table> -->
```
Comment (garry911128) [inline Html_File/AccountSetting.html:94]: What's happening here?

**Comment by Vghxv** [comment]
is this account setting page a draft? password should not be visible

File: Html_File/AccountSetting.html (hunk 1 above)
Comment (cuchaos) [reply Html_File/AccountSetting.html]: forget delete, now delete

**Review by garry911128 — APPROVED** [review]

**Comment by cuchaos** [comment]
> is this account setting page a draft? password should not be visible

now password input is not visible

**Review by Vghxv — APPROVED** [review]

**Pull request merged**